src/components/containers/seo.tsx

```diff
@@ -249,8 +194,28 @@ const SEO = ({
                             />
                         )
                     })}
-        </Helmet>
+        </>
     )
 }
 
 export default SEO
+
+// Since Gatsby Head Api dose not support dynamic body dir I wrote this!
+// In the future we can replace this with sth better!
+// We need this for our RTL and LTR
+export const PageDirection = () => {
```

Comment:
As I mentioned in the comment we need this for our page directions, later in the future we may replace this with sth better!